pycode/memilio-epidata/memilio/epidata_test/test_epidata_get_vaccination_data.py

```diff
@@ -175,7 +175,6 @@ def test_download_not_working(self, mock_download, mock_print):
             "Error in reading csv while downloading vaccination data.")
         mock_print.assert_has_calls([expected_call])
 
-
     def test_interval_mapping(self):
```

Comment:
this and other tests regarding general functionality should also be removed from test_vaccination_data and added to a suited test file!

Reply:
test moved into `test_epidata_modifyDataframeSeries.py`